src/Downloader.php

```diff
@@ -412,4 +412,11 @@ private function initialize()
             $this->endpoints[$endpoint] = $config;
         }
     }
+
+    private static function generateCacheKey(string $url): string
+    {
+        $url = preg_replace('@^https://api.github.com/repos/symfony/recipes(-contrib)?/contents/@', 'sf-recipes$1/', $url);
```

Comment:
I'd suggest decoupling from symfony's repo, aka use this instead:
`$url = preg_replace('~^https://api.github.com/repos/([^/]++/[^/]++)/contents/', '$1/', $url);`